### PR TITLE
chore: bump OTP version

### PR DESCRIPTION
**Pull request by lafirest**
upgrade the OTP version for security reasons

## PR Checklist
Please convert it to a draft if any of the following conditions are not met. Reviewers may skip over until all the items are checked:

- [x] For internal contributor: there is a jira ticket to track this change, and another jira tickt to track doc updates (if any)
       EMQX-7683
- [x] In case of non-backward compatible changes, reviewer should check this item as a write-off, and add details in **Backward Compatibility** section

### CODE REVIEW

File: scripts/ct/run.sh [hunk 1]
```diff
@@ -98,7 +98,7 @@ TTY=''
 if [[ -t 1 ]]; then
     TTY='-t'
 fi
-docker exec -i $TTY "$ERLANG_CONTAINER" bash -c 'git config --global --add safe.directory /emqx'
+docker exec -i $TTY "$ERLANG_CONTAINER" bash -c 'git config --global --add safe.directory "*"'
```
Comment (thalesmg) [inline scripts/ct/run.sh:101]: Just confirming: is this change intended here?  It might have different behaviors depending on the work dir.

**Review by thalesmg — APPROVED** [review]

File: scripts/ct/run.sh (hunk 1 above)
Comment (id) [reply scripts/ct/run.sh:101]: +1 to Thales, what is the reason for this change?

Comment (lafirest) [reply scripts/ct/run.sh:101]: Without this change, some actions won't work successfully (authn/authz), and it will report some repos are unsafe (like quic), even if I add these repos one by one, these actions still won't succeed.

Comment (id) [reply scripts/ct/run.sh]: @lafirest another option is to use the same user and group id when building inside the container
Haven't looked what are corresponding compose options, but this one worked for me (running make inside emqx-builder):
```
-v ~/.cache:/.cache -u $(id -u $USER):$(id -g $USER)
```

**Review by id — APPROVED** [review]
